.github/workflows/e2e-tests.yaml

```diff
@@ -18,7 +18,13 @@ jobs:
     strategy:
       max-parallel: 99
       matrix:
-        e2e-test: ["ring", "basic_aggregator", "model_aggregator"]
+        e2e-test:
+          [
+            "ring",
+            "basic_aggregator",
+            "model_aggregator",
```

Comment:
MAYBE: in a future PR, we could switch this to 
pretrained_model_aggregator
the current test to just model_aggregator to be explicit with repo names ?
cc: @shubham3121  @khoaguin

Reply:
I'd like to have `pretrained_model_aggregator` and `model_aggregator_with_local_training` so the names are explicit in what the test does, since we have 2 repos for each test